### ISSUE TITLE
Hostile Remover breaking API7 1.12.2 DW20

### ISSUE THREAD

**Issue by SeanStoves** [Time6628/CatClearLag#19]
We are seeing the below console spam whenever the ClearLag runs.

```
[13:25:34] [pool-2-thread-37/INFO] [clearmob]: 0 Entity were removed
>
[13:25:34] [pool-2-thread-37/INFO] [clearmob]: 0 Tile Entity were removed
> [13:25:34] [pool-2-thread-37/INFO]: [ClearMob] Entities have been cleared
>
[13:29:51] [pool-2-thread-39/ERROR] [Sponge]: The Scheduler tried to run the task CatClearLag hostile checker owned by Plugin{id=catclearlag, name=CatClearLag, version=0.8.1, description=A plugin to assist in removing lag from your server., source=/home/container/./mods/plugins/CatClearLag-0.8.1.jar}, but an error occured.
java.lang.NullPointerException: null
at java.util.Objects.requireNonNull(Objects.java:203) ~[?:1.8.0_161]
at java.util.function.Predicate.and(Predicate.java:68) ~[?:1.8.0_161]
at me.time6628.clag.sponge.EntityRemover.<init>(EntityRemover.java:40) ~[EntityRemover.class:?]
at me.time6628.clag.sponge.CatClearLag.getXPOrbs(CatClearLag.java:180) ~[CatClearLag.class:?]
at me.time6628.clag.sponge.runnables.EntityChecker.run(EntityChecker.java:20) ~[EntityChecker.class:?]
at org.spongepowered.api.scheduler.Task$Builder.lambda$execute$0(Task.java:139) ~[Task$Builder.class:1.12.2-2611-7.1.0-BETA-2970]
at org.spongepowered.common.scheduler.SchedulerBase.lambda$startTask$0(SchedulerBase.java:186) ~[SchedulerBase.class:1.12.2-2611-7.1.0-BETA-2970]
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_161]
at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_161]
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_161]
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_161]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_161]
```

Any help would be helpful.  

```
SpongeForge
Minecraft: 1.12.2
SpongeAPI: UNKNOWN
SpongeForge: UNKNOWN
Minecraft Forge: 14.23.2.2618
>
```

```
CatClearLag-0.8.1.jar
spongeforge-1.12.2-2611-7.1.0-BETA-2970.jar
SpongeBootstrap-0.6.2.jar
```

**Comment by Cmoor** [comment]
I'd like to confirm that I too am getting this error in console on our 1.12.2 FTB Revelation server.
CatCLearLag: 0.8.1
Spongeforge: 1.12.2-2611-7.1.0-BETA-2931
Forge: 1.12.2-14.23.2.2618

https://hastebin.com/yumoweruxe.vbs

EDIT: I also found that it is only towards XP orbs. I tried the command /re hostiles and no error. I used /re xp and I was given an NPE and an error saying "Error occurred while executing command: null"
